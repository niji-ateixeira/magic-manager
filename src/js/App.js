import React from 'react';
import { Header, Footer } from './view/containers';

const App = () => (
  <div>
    <Header />
    <Footer />
  </div>
);

export default App;
