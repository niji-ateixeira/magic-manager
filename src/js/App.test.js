import React from 'react';
import renderer from 'react-test-renderer';
import App from './App.js';

test('renders without crashing', () => {
  const component = renderer.create(
    <App />,
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
