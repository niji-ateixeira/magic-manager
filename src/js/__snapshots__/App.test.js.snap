// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App-App-0-1-1"
>
  <header>
    Header
  </header>
  <div>
    Que ce passe-t-il dans ta tete?
  </div>
  <footer>
    Footer
  </footer>
</div>
`;
